NuRadioReco/modules/RNO_G/beam_pattern.py

```diff
@@ -0,0 +1,71 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from NuRadioReco.utilities import units
+from NuRadioReco.modules import channelAddCableDelay, channelBandPassFilter, sphericalWaveFitter
+from NuRadioReco.detector import detector
+from NuRadioReco.modules.io.RNO_G import readRNOGDataMattak
+from datetime import datetime
+import NuRadioReco.detector.detector
+import astropy.time
+import logging
+from NuRadioReco.utilities import fft
+from os import listdir
+from os.path import isfile, join
+from scipy.signal import correlate
+from NuRadioReco.utilities.fft import time2freq
+from NuRadioReco.utilities.fft import freq2time
+from scipy.interpolate import interp1d
+import pandas as pd
+import json
+
+#define json file to take positions from
+distance_params_df = pd.DataFrame(list(json.load(open('/home/sanyukta/astropart/data/json/RNO_season_2023.json'))['channels'].values()))
+
+def get_channel_positions(station_number):
+    """
+    To find the position of each channel in a given year (using json file)
+    Input parameters: 
+    -station number (should be a valid station in the json file)
+    Returns:
+    - 3 arrays of x, y, z coordinates of each channel
+    """
+    x, y, z = {}, {}, {}
+    for ch in [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 21, 22, 23]:
```

Comment:
I would not necessarily expect this function to return only a subset of channels

Reply:
Yes I did it only for the deep channels